auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -14,41 +14,80 @@
                   My Business Registry</a>
                 page and use the Name Request Number to:
               </v-list-item-subtitle>
-              <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index">
-                <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
-                <v-list-item-content>
-                  <v-list-item-subtitle class="list-item-text">
-                    {{item.text}}
-                  </v-list-item-subtitle>
-                </v-list-item-content>
-              </v-list-item>
+              <div v-if="enableBcCccUlc">
```

Comment:
can you please add a computed value for `bulletPoints` that changes based on the `enableBcCccUlc` flag instead of having the v-if/else clause?